### PR TITLE
Add permissions section to MembersInfoPopover

### LINKED ISSUE #2658: Add permissions section to MemberInfo
### Description

Implement a section on MemberInfo that shows what permissions does the user has. (This includes both the styles and the logic to get the user's permissions).

**Questions for Product:**

1. How does this section should look like if the user doesn't have any permissions?
   R: If the user doesn't have any permissions then we should hide this section.

FireShot Capture 426 - Colony Simplified – Figma - www figma com

Figma file: https://www.figma.com/file/qQoBbyyDXkuHwoqHIOGjEu/Colony-Simplified?node-id=422%3A10616

### PR DESCRIPTION
**Pull request by ArmandoGraterol**
Add user permissions section to MembersInfoPopover. If the user doesn't have any permissions, then the section shouldn't appear

FireShot Capture 429 - Colony - localhost

Resolves #2658

### CODE REVIEW

**Review by alicjakujawa — COMMENTED** [review]
Screenshot 2021-09-02 at 13 37 27
not sure if im testing something wrong, but im creator of this colony, so i have all permissions and nothing is shown here

**Comment by ArmandoGraterol** [comment]
@alicjakujawa are you sure that you are on the branch of this PR? Because from the screenshot it seems like you have a token section, which should only be available on #2679, and this is #2678

**Review by chinins — APPROVED** [review]
Looks good! Working fine for me:

Screenshot 2021-09-02 at 20 03 54

File: src/modules/core/components/InfoPopover/MemberInfoPopover/UserPermissions.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+const UserPermissions = ({ roles }: Props) => {
+  return (
+    <div className={styles.permissionsContainer}>
+      <Heading
+        appearance={{
+          size: 'normal',
+          margin: 'none',
+          theme: 'grey',
+          weight: 'bold',
+        }}
```
Comment (rdig) [inline src/modules/core/components/InfoPopover/MemberInfoPopover/UserPermissions.tsx:32]: So I know this has the `theme: grey` set, but for me it still renders as var `--dark`

Screenshot from 2021-09-02 22-08-43
Screenshot from 2021-09-02 22-09-59

Please investigate, as this is the only thing that's not according to spec

File: src/modules/core/components/InfoPopover/MemberInfoPopover/UserPermissions.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    defaultMessage: 'Permissions',
+  },
+});
+
+const UserPermissions = ({ roles }: Props) => {
+  return (
+    <div className={styles.permissionsContainer}>
+      <Heading
+        appearance={{
+          size: 'normal',
```
Comment (rdig) [inline src/modules/core/components/InfoPopover/MemberInfoPopover/UserPermissions.tsx:28]: Also, it looks to me like the font size is smaller in this screenshot:

Screenshot from 2021-09-02 22-13-31

Please compare with the Figma file and adjust it accordingly

**Review by rdig — APPROVED** [review]
Except the section heading color and size, everything else is good.

I'm pre-approving it, but please fix those issues before merging.

File: src/modules/users/components/UserInfo/UserInfo.tsx [hunk 3]
```diff
@@ -45,22 +45,22 @@ const defaultRenderAvatar = (
...
-  colonyAddress: Address;
+  colony: Colony;
   placeholder?: MessageDescriptor;
   user?: AnyUser;
   userAddress?: Address;
   renderAvatar?: (
     address: Address,
     user?: AnyUser,
-    colonyAddress?: Address,
+    colonyAddress?: Colony,
```
[the remark this answers is not in the document]
Comment (ArmandoGraterol) [reply src/modules/users/components/UserInfo/UserInfo.tsx:55]: Correct! This one must have slipped my mind, thanks

**Pull request merged**